.github/workflows/publish-container.yml

```diff
@@ -0,0 +1,58 @@
+#
+name: Create and publish the PyStemmusScope container image 📦
+
+on:
+  release:
+    types: [published]
+
```

Comment:
You could add a manual trigger here. This can be useful if the build failed for some reason and you want to trigger this action through the github interface

Reply:
Action looks fine as far as I can see otherwise :+1: